bodhi/server/consumers/updates.py

```diff
@@ -55,7 +54,8 @@ class UpdatesHandler(fedmsg.consumers.FedmsgConsumer):
     config_key = 'updates_handler'
 
     def __init__(self, hub, *args, **kwargs):
-        self.db_factory = models.get_db_factory()
+        engine = initialize_db(config)
+        self.db_factory = util.transactional_session_maker(engine)
 
         prefix = hub.config.get('topic_prefix')
         env = hub.config.get('environment')
```

Comment:
I don't think this pull request *has* to be the one to deal with it, but there are still other consumers that are creating their own DB connections their own way. ```bodhi.server.consumers.masher``` and ```bodhi.server.consumers.signed``` are both creating connections their own way.